homeassistant/components/tautulli/strings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Tautulli",
+        "description": "To find your API key, open the Tautulli web interface and navigate to Settings and then to Web interface. The API key will be at the bottom of that page.\n\nExample of the URL: http://192.168.0.10:8181 with 8181 being the default port.",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key]",
+          "url": "[%key:common::config_flow::data::url%]",
+          "verify_ssl": "[%key:common::config_flow::data::verify_ssl]"
+        }
+      },
+      "reauth_confirm": {
+        "title": "Tautulli",
+        "description": "To find your API key, open the Tautulli web interface and navigate to Settings and then to Web interface. The API key will be at the bottom of that page.",
```

Comment:
Should we tell we are re-authenticating here? As in, mention it in some way?

Is it "Web interface" or "web interface"? Both variants are in this line.